### DISCUSSION TITLE
How to get cell ids list?

### DISCUSSION THREAD

**Discussion by zxcvqwerasdf** [SteamRE/SteamKit#1213] [General]
Once it was possible to find out the names of the regions of cells through https://github.com/SteamDatabase/SteamTracking/blob/master/ClientExtracted/steam/cached/CellMap.vdf
But now this file is don't exist
How can you now find out the names of the regions for each cell id?
For requests like
https://api.steampowered.com/ISteamDirectory/GetCMList/v1/?cellid=0
Thank you

**Comment by bbergeron0** [comment]
For posterity, here's that file last version before it got deleted: https://github.com/SteamDatabase/SteamTracking/blob/992bc2f5cdc2d011b3e273c61c7836d2ed2bb46f/ClientExtracted/steam/cached/CellMap.vdf
